manifest.json

```diff
@@ -23,6 +23,53 @@
     ],
     "dependencies": { "language_c": {}, "lsp": {} }
   },{
+  "addons": [{
+    "id": "lsp_clojure",
+    "description": "Automatic configuration/binary download for LSP completion for clojure with clojure-lsp.",
+    "path": "plugins/lsp_clojure.lua",
+    "version": "2024.03.31-19.10.13",
```

Comment:
We don't support dashes in the version.
It's probably fine to just drop the dash and anything after it.